### PR TITLE
Update: Korean translation up to 0.59.0

### PR DESCRIPTION
**Pull request by telk5093**
:)

### CODE REVIEW

**Comment by JGRennison** [comment]
Incidentally, I've added a couple of extra texts in the last commit. Could be worth adding to this PR.

**Comment by telk5093** [comment]
Thanks for notice, updated.

File: src/lang/extra/korean.txt [hunk 1]
```diff
@@ -90,11 +95,16 @@ STR_GRAPH_DELIVERED_CARGO_BY_CARGO_MODE                         :{TINY_FONT}{BLA
...
 
+STR_GRAPH_LAST_72_PRODUCTION_INTERVALS_TIME_LABEL               :{TINY_FONT}{BLACK}Last 72 production intervals
+
 STR_SMALLMAP_SCREENSHOT                                         :{BLACK}스크린샷
 
 STR_NEWS_VEHICLE_NO_DEPOT_ORDER                                 :{WHITE}{VEHICLE}의 경로에는 기지가 존재하지 않습니다
 STR_NEWS_TRAIN_OVERSHOT_STATION                                 :{WHITE}{VEHICLE} : 속력이 너무 빨라서 {STRING}에 멈추지 못했습니다
 
+STR_NEWS_ROAD_REBUILDING_PERIODS                                :{BIG_FONT}{BLACK}Traffic chaos in {TOWN}!{}{}Road rebuilding programme funded by {STRING} brings half a period of misery to motorists!
+STR_NEWS_EXCLUSIVE_RIGHTS_DESCRIPTION_PERIOD                    :{BIG_FONT}{BLACK}Local authority of {TOWN} signs contract with {STRING} for 1 period of exclusive transport rights!
```
Comment (JGRennison) [inline src/lang/extra/korean.txt:106]: This looks to not be translated yet?

File: src/lang/extra/korean.txt [hunk 2]
```diff
@@ -90,11 +95,16 @@ STR_GRAPH_DELIVERED_CARGO_BY_CARGO_MODE                         :{TINY_FONT}{BLA
 STR_GRAPH_DELIVERED_CARGO_BY_COMPANY_MODE_TOOLTIP               :{BLACK}선택한 회사를 각각 하나의 선으로 표시합니다
 STR_GRAPH_DELIVERED_CARGO_BY_CARGO_MODE_TOOLTIP                 :{BLACK}선택한 화물를 각각 하나의 선으로 표시합니다
 
+STR_GRAPH_LAST_72_PRODUCTION_INTERVALS_TIME_LABEL               :{TINY_FONT}{BLACK}Last 72 production intervals
```
Comment (JGRennison) [inline src/lang/extra/korean.txt:98]: Likewise here

File: src/lang/extra/korean.txt (hunk 1 above)
Comment (telk5093) [reply src/lang/extra/korean.txt:106]: Oops :P

**Pull request merged**